src/vk/VkCommands.cs

```diff
@@ -0,0 +1,108 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Vulkan
+{
+    public static unsafe partial class VulkanNative
+    {
+        public static VkResult vkQueueSubmit(VkQueue queue, params VkSubmitInfo[] submits)
```

Comment:
I appreciate the contribution, but these are the sorts of overloads I would like to leave out of these low-level bindings. Anything added that isn't a direct mapping with a regular Vulkan function just adds mental overhead, IMO, and diverges the library further from the rest of the Vulkan ecosystem.

Arguably, this only makes it slightly easier to use the function with arrays anyways. Instead of doing this, you can just call one of the existing overloads and pass a reference to the first element of the array.